news/plot_bug.rst

```diff
@@ -0,0 +1,11 @@
+**Added:** None
+
+**Changed:** plotter.py
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:** #114
```

Comment:
This is for user-facing documentation, so it should say something like "fixed plotting bug in GUI interface" and should be formatted as follows:
```
**Fixed:**
* plotting bug in GUI interface
```
This will then build as a bulleted list.